openshift-knative-operator/hack/update-manifests.sh

```diff
@@ -45,7 +45,7 @@ function download_serving {
     target_file="$target_dir/$index-$file"
 
     if [[ ${KNATIVE_SERVING_MANIFESTS_DIR} = "" ]]; then
-      url="https://raw.githubusercontent.com/openshift-knative/serving/${branch}/openshift/release/artifacts/$index-$file"
+      url="https://raw.githubusercontent.com/openshift-knative/serving/${branch}/openshift/release/artifacts/$file"
```

Comment:
So we stop downloading the files with index and get the other ones?

Reply:
There is no more index in the files in Serving from 1.7+.